src/test/java/redis/clients/jedis/tests/commands/ObjectCommandsTest.java

```diff
@@ -1,15 +1,42 @@
 package redis.clients.jedis.tests.commands;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
+import redis.clients.jedis.Jedis;
 import redis.clients.jedis.util.SafeEncoder;
 
+import java.util.List;
+
 public class ObjectCommandsTest extends JedisCommandTestBase {
 
   private String key = "mylist";
   private byte[] binaryKey = SafeEncoder.encode(key);
+  private int port = 6386;
```

Comment:
Please read/set the as it was done in other code. This port was probably used in migrate related tests.

Reply:
I created a new Redis instance for lfu test case.

Reply:
Would you be able to reuse an existing one?